threedi_api_qgis_client/widgets/simulation_init.py

```diff
@@ -0,0 +1,113 @@
+import os
+from qgis.PyQt import uic
+from qgis.PyQt.QtCore import Qt, QSettings, QThread
+from qgis.PyQt.QtGui import QStandardItemModel, QStandardItem
+from qgis.PyQt.QtWidgets import QFileDialog
+
+base_dir = os.path.dirname(os.path.dirname(__file__))
+uicls, basecls = uic.loadUiType(os.path.join(base_dir, 'ui', 'init_dialog.ui'))
+
+
+class SimulationInit(uicls, basecls):
+    """Dialog with methods for handling running simulations."""
+    PROGRESS_COLUMN_IDX = 2
+
+    def __init__(self, parent_dock, parent=None):
+        super().__init__(parent)
+        self.setupUi(self)
+        self.parent_dock = parent_dock
+
+        self.open_wizard = False
+        self.initial_conditions = None
+        self.multiple_simulations_widget.setVisible(False)
+        self.load_state_widget.setVisible(False)
+        self.damage_estimation_widget.setVisible(False)
+        self.postprocessing_widget.setVisible(False)
+        self.cb_multiple_simulations.stateChanged.connect(self.multiple_simulations_changed)
+        self.cb_conditions.stateChanged.connect(self.load_saved_state_changed)
+        self.cb_postprocess.stateChanged.connect(self.postprocessing_state_changed)
+        self.cb_damage_estimation.stateChanged.connect(self.damage_estimation_changed)
+        self.pb_next.clicked.connect(self.start_wizard)
+        self.pb_cancel.clicked.connect(self.close)
+        self.fill_comboboxes()
+
+    def fill_comboboxes(self):
+        self.dd_cost_type.addItems(["min", "avg", "max"])
+        self.dd_flood_month.addItems(["jan", "feb", "mar", "apr", "may", "jun", "jul", "aug", "sep", "oct", "nov", "dec"])
+        self.dd_number_of_simulation.addItems([str(i) for i in range(2, 10)])
+        self.dd_simulation_difference.addItems(["precipitation", "breaches"])
+
+    def multiple_simulations_changed(self, int):
+        if int:
+            self.multiple_simulations_widget.show()
+        else:
+            self.multiple_simulations_widget.hide()
+
+    def load_saved_state_changed(self, int):
+        if int:
+            self.load_state_widget.show()
+        else:
+            self.load_state_widget.hide()
+
+    def postprocessing_state_changed(self, int):
```

Comment:
Please change argument name from 'int' to something that will no shadow built-in funtion.